docs/source/sensors.rst

```diff
@@ -125,7 +125,7 @@ Timer sensor
 ^^^^^^^^^^^^
 
 Look at the timer sensor implementation
-`here <https://github.com/StackStorm/st2/blob/master/st2reactor/st2reactor/contrib/sensors/st2_timer_sensor.py>`__.
+`here <https://github.com/StackStorm/st2/blob/master/st2reactor/st2reactor/timer/base.py>`__.
```

Comment:
I saw your Slack comment - I'm on it, but next time we should try to be more complete and also update docs when the code changes are made :P